src/main/java/edu/ucsb/nceas/mdqengine/scheduler/RequestReportJob.java

```diff
@@ -610,62 +621,273 @@ public void submitReportRequest(MultipartCNode cnNode, MultipartMNode mnNode, Bo
         SystemMetadata sysmeta = null;
         InputStream objectIS = null;
         InputStream runResultIS = null;
+        HashStore hashStore = null;
 
         Identifier pid = new Identifier();
         pid.setValue(pidStr);
 
         try {
-            if (isCN) {
-                sysmeta = cnNode.getSystemMetadata(session, pid);
-            } else {
-                sysmeta = mnNode.getSystemMetadata(session, pid);
-            }
-        } catch (NotAuthorized na) {
-            log.error("Not authorized to read sysmeta for pid: " + pid.getValue() + ", continuing with next pid...");
-            return;
+            hashStore = getHashStoreFromMetadigProps();
+
         } catch (Exception e) {
-            throw (e);
+            log.warn("HashStore is unavailable. Additional info: " + e.getMessage());
         }
 
-        try {
-            if (isCN) {
-                objectIS = cnNode.get(session, pid);
-            } else {
-                objectIS = mnNode.get(session, pid);
+        // Retrieve the system metadata
+        if (hashStore != null) {
+            try {
+                sysmeta = getSystemMetadataFromHashStore(pid, hashStore);
+            } catch (Exception e) {
+                // Attempt to retrieve sysmeta from the API as a backup
+                sysmeta = getSystemMetadataFromMnOrCn(pid, cnNode, mnNode, isCN, session, sysmeta);
             }
-            log.trace("Retrieved metadata object for pid: " + pidStr);
-        } catch (NotAuthorized na) {
-            log.error("Not authorized to read pid: " + pid
-                    + ", unable to retrieve metadata, continuing with next pid...");
-            return;
+        } else {
+            sysmeta = getSystemMetadataFromMnOrCn(pid, cnNode, mnNode, isCN, session, sysmeta);
+        }
+
+        // Retrieve the EML metadata document for the given pid
+        if (hashStore != null) {
+            try {
+                objectIS = getEMLMetadataDocFromHashStore(pid, hashStore);
```

Comment:
please rename this method to just `getObjectFromHashStore` or something similar. metadig runs on lots of dialects, not just EML, and at some point this might be repurposed to get data objects in addition to metadata objects